bindings/python/generate_bindings.py

```diff
@@ -36,71 +110,86 @@ def handle_final(filename, module_name):
     return linelist
 
 
-def handle_include(item):
+def handle_CALL_EXPR(item):
     pass
 
 
-def handle_alias(item):
+def handle_NAMESPACE_REF(item):
     pass
 
 
-def handle_constructor(item):
-    parameters_type = ""
-    parameters_type = ",".join(params for params in item["parameters_type"])
-    module_linelist.append(f".def(py::init<{parameters_type}>())")
+def handle_TYPE_REF(item):
+    global TYPE_REF_LIST
+    TYPE_REF = item["name"]
+    if TYPE_REF_LIST[-1][0]:
+        TYPE_REF_LIST[-1][1] = TYPE_REF
+    else:
+        TYPE_REF_LIST.append([None, TYPE_REF])
+
+
+def handle_PARM_DECL(item):
+    for sub_item in item["members"]:
+        kind_functions[sub_item["kind"]](sub_item)
```

Comment:
Where is `kind_functions` at global scope?

Reply:
Changed to a class member